appengine/flexible/tasks/README.md

```diff
@@ -0,0 +1,105 @@
+# Google Cloud Tasks App Engine Queue Samples
+
+Sample command-line program for interacting with the Cloud Tasks API
+using App Engine queues.
+
+App Engine queues push tasks to an App Engine HTTP target. This directory
+contains both the App Engine app to deploy, as well as the snippets to run
+locally to push tasks to it, which could also be called on App Engine.
+
+`app_engine_queue_snippets.py` is a simple command-line program to create tasks
+to be pushed to the App Engine app.
+
+`main.py` is the main App Engine app. This app serves as an endpoint to receive
+App Engine task attempts.
+
+`app.yaml` configures the App Engine app.
+
+
+## Prerequisites to run locally:
+
+Please refer to [Setting Up a Python Development Environment](https://cloud.google.com/python/setup).
+
+## Authentication
+
+To set up authentication locally, download the
+[Cloud SDK](https://cloud.google.com/sdk), and run
+
+    gcloud auth application-default login
+
+On App Engine, authentication credentials will be automatically detected.
+
+On Compute Engine and Container Engine, authentication credentials will be
```

Comment:
Are these necessary? Isn't the tutorial for App Engine queues targeted at App Engine endpoints?

Reply:
As above